File: boyoung_chae/objective_c_examples/PropertiesModifiers_Person/PropertiesModifiers_Person/Person.m
//
//  Person.m
//  PropertiesModifiers_Person
//
//  Created by Boyoung on 2016-09-08.
//  Copyright © 2016 Boyoung. All rights reserved.
//

#import "Person.h"

@implementation Person

- (void) setTheValue:(NSString *)nationality
{
    _nationality = nationality;
    _identity = 123;
}

// NSInteger is not an object. (@ is not used.)
// NSNumber is an object. (@ is used.)

- (NSString *)description
{
    return [NSString stringWithFormat:@"\nName: %@, ID: %lu, Nationality: %@", self.name, self.identity, self.nationality];
}

@end

File: olderFiles/boyoung_chae/c_examples/PetShop/PetShop/Person.c
//
//  Person.c
//  PetShop
//
//  Created by Boyoung on 2016-09-02.
//  Copyright © 2016 Boyoung. All rights reserved.
//

#include "Person.h"

Person* Person_new()
{
    return malloc(sizeof(Person));
}